### PR TITLE
chore: refine updating message endpoint for local debug

### PR DESCRIPTION
**Pull request by IvanJobs**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/15371383
1. Make error message more specific.
2. Add checking steps in bot-help.md.
3. Fix duplicated error messages.

Found a new bug of duplicated error messages:
image

Fixed:
image

### CODE REVIEW

**Review by MuyangAmigo — APPROVED** [review]
LGTM

File: docs/fx-core/bot-help.md [hunk 1]
```diff
@@ -8,6 +8,25 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
...
 
+## BT.BotRegistrationNotFoundError
+
+### Error Message
+
+Bot registration was not found. Click 'Get Help' button to learn more about how to check bot registrations.
+
+### Mitigation
+
+The first step is to make sure the remote bot registration is not existing.
```
Comment (eriolchan) [inline docs/fx-core/bot-help.md:19]: Maybe "is not existing" -> "doesn't exist"?

File: docs/fx-core/bot-help.md [hunk 2]
```diff
@@ -8,6 +8,25 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
...
+Suppose your Teams app's name is `someTeamsApp`, then the target bot registration name you are supposed to find is something like `someTeamsApp-local-debug`. So follow the steps below:
+1. Login to [Bot Framework Dev Portal](https://dev.botframework.com/) with your M365 account which is the same account you used in Teams Toolkit.
```
Comment (eriolchan) [inline docs/fx-core/bot-help.md:21]: In our document, we need to apply and align the terminology, making "M365" -> "Microsoft 365"

File: docs/fx-core/bot-help.md [hunk 3]
```diff
@@ -8,6 +8,25 @@ The subscription didn't register to use namespace 'Microsoft.BotService'.
...
+1. Click `My Bots` to check the list of bot registrations associated with the current M365 account.
```
Comment (eriolchan) [inline docs/fx-core/bot-help.md:22]: Same here

**Review by eriolchan — APPROVED** [review]

**Pull request merged**